waterlog/src/reducers/rootReducer.js

```diff
@@ -54,9 +54,10 @@ const rootReducer = combineReducers({
   forecastMonthly:CostsMonthlyForecastReducer,
   dailytankgraph:TankGraphReducer,
   heatMap:HeatMapsReducer,
+  heatMap:HeatMapsReducer,
+  placeholder:ForecastPlaceholderReducer,
   numNotifs : NotificationsReducer,
   filteredAlerts: FilterReducer,
   dailytankgraph: TankGraphReducer,
-  placeholder: ForecastPlaceholderReducer
```

Comment:
if this line is removed, remove floating comma in the above line.

Reply:
done

Reply:
done